src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: { categories: [] },
+  reducers: {
+    check: () => 'Under construction',
+  },
```

Comment:
- [ ]  You should define CHECK_STATUS as action type following the [ducks pattern](https://github.com/erikras/ducks-modular-redux)